docs/concept-exercises.md

```diff
@@ -75,7 +75,7 @@ Each task must conform to the following standard:
 - Start with a third-level heading starting with a number (.e.g `### 1.`, `### 2.`)
 - Describe which function/method the student needs to define/implement (e.g. `Implement method X(...)` that takes an A and returns a Z),
 - Provide an example usage of that function. These examples should be different examples to those given in the tests.
-
+- We place high value on making Exercism’s content safe for everyone and so often err on the side of caution in deciding whether stories are appropriate or not. If you are unsure if something might work or not, please mention @exercism/leadership and we can clarify.
```

Comment:
What about:

```suggestion
- The contents must adhere to Exercism's [Code of Conduct](https://exercism.io/code-of-conduct), as we place high value on making Exercism’s content safe and inclusive for everyone. Feel free to mention @exercism/leadership if you have any questions or would like some help.
```

Reply:
I don’t feel like this has anything to do with CoC though. The story about weighing yourself wasn’t in any way a violation of the CoC and it wasn’t offensive in any way. But for some people it could be triggering. 

The point here really is that this is Exercism's content, rather than a person's interactions within Exercism. So we're going to police the stories etc.

Reply:
Right. I still feel that we're making for too big a deal of this :)

Reply:
I'd still like to shorten it. Maybe add something about us looking at it together?

```suggestion
- We place high value on making Exercism’s content safe for everyone. If you'd like to check a story with us, please mention @exercism/leadership and we'll look at it together.
```

By the way, the heading is "Each task must conform to the following standard:", which does not lead nicely into "We place high value on making" as this is not just about the tasks, but also about the story part. As such, maybe we should make it not a bullet point but a separate bit of text.